### ISSUE TITLE
martinlabs-commons dependency compilation errors, missing info how to run everything

### ISSUE THREAD

**Issue by smxsm** [MartinLabs/generator-martinlabs#1]
I've tried to generate an app, but the Java part is throwing lots of compilation errors currently, e.g. the "DaoWrapper" constructors seem to have changed, there is no "sha256" method anymore, same goes for "OauthHelper" etc. 

Also, it is a bit unclear how to run the frontend and backend parts, that seems to be missing in the README and in the pom.xml etc.? I was able to get the frontend running by extending the package.json and webpack config using the **"webpack-dev-server"**, for the backend I've added **jetty** as dependency/plugin to the pom.xml ... but then the various compilation errors struck me and I gave up for now ... trying JHipster now instead :)

**Comment by melanke** [comment]
Yes, I am sorry about that, you know, is a pain to update maven repositories.

Sadly JHipster isn't a CRUD generator based on your database, but we really need to fix this issues before you can use ours.

I will try to fix this as soon as possible and answer this issue to let you know

Thanks for trying :/
